### DISCUSSION TITLE
node.activateOTAA(0) as923, error code -1114

### DISCUSSION THREAD

**Discussion by philhuen** [jgromes/RadioLib#1263] [General]
Any one tried to use AS923 + node.activateOTAA(0) with success? 
 node.activateOTAA(2) is ok, however is there a way to disable DWELL before node.activateOTAA(0)?

**Comment by StevenCellist** [comment]
Well, there are laws and regulations that forbid users from exceeding dwell time limitations that are supposed to be in place. And the Regional Parameters v1.0.4 enforce dwell time for AS923, subsequently prohibiting the use of DR0 and DR1 for JoinRequests since the packet size exceeds the dwell time.
TL/DR; I recommend you stay legal and stick to the dwell time limitations!
